packages/wallets/src/components/WalletCard/WalletCard.tsx

```diff
@@ -26,15 +37,29 @@ const WalletCard: React.FC<TProps> = ({ balance, currency, iconSize = 'lg', isDe
                 isDemo={isDemo}
                 type='card'
             >
-                <div className='wallets-card__details'>
-                    <div className='wallets-card__details__top'>
+                <div
+                    className={classNames('wallets-card__details', {
+                        'wallets-card__details--carousel-content': isCarouselContent,
+                        'wallets-card__details--carousel-content-active': isCarouselContent && isActive,
```

Comment:
nit/minor: --modifiers are typically separated by two dashes, .e.g --active not -active

Reply:
Updated thanks